make-containerized-build.groovy

```diff
@@ -28,7 +28,7 @@ node('cico-workspace') {
 	try {
 		stage('prepare bare-metal machine') {
 			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh root@${CICO_NODE}:'
-			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build --gitrepo=${GIT_REPO} --branch=${GIT_BRANCH}'
+			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=tip/ --gitrepo=${GIT_REPO} --ref=pull/${ghprbPullId}/head'
```

Comment:
use `/opt/build` as directory (or better, `/opt/build/go/src/github.com/ceph/ceph-csi`), otherwise it lands in some local place (probably `/root/tip`). If we want to use standard go tools, the directory needs to be in the `GOPATH` format.

Reply:
There needs to be a check that `ghprbPullId` is available as environment  variable, if it is not, fall back to `--ref=master`.
Time to learn some Groovy :disappointed: